Add support for metadata in CBS (#4581)
kyma-project/kyma: docs/asset-store/06-01-asset.md

| **metadata.name** |    **YES**   | Specifies the name of the CR. |
| **metadata.namespace** |    **YES**   | Defines the Namespace in which the CR is available. |
| **spec.source.mode** |    **YES**   | Specifies if the asset consists of one file or a set of compressed files in the ZIP or TAR formats. Use `single` for one file and `package` for a set of files. |

> | **spec.source.metadata** |    **NO**   | Specifies the set of the metadata for Asset. It must be defined in a valid YAML/JSON format. |

Reviewer: ```suggestion
| **spec.source.metadata** |    **NO**   | Specifies a set of metadata for the Asset. Define it in a valid YAML or JSON format. |
```
